utils/Unpatching.py

```diff
@@ -165,7 +165,8 @@ def get_first_index(iX, iY, iZ, patch_nmb_layer, x_index, y_index):
 
     return num
 
-def fRigidUnpatchingCorrection(actual_size, allPatches, patchOverlap):
+def fRigidUnpatchingCorrection(actual_size, allPatches, patchOverlap, mode='overwritten'):
+    allPatches = 255 * allPatches

```

Comment:
fixed scaling to 255? -> Remove!
Why is scaling prior to unpatching required? 
Maybe make it as a parameter and correct scaling to (`img=allPatches` and `range = [0 255]`):
`scaledImg = ((img - min(img(:))) * (range(2)-range(1)))./(max(img(:)) - min(img(:)));`